docs/en/new-media.md

```diff
@@ -514,9 +514,9 @@ Edition
 
 ### Topics
 
-<RouteEn author="emdoe" example="/nautilus/topic/arts" path="/nautilus/topic/:tid" :paramsDesc="['topic']">
+<RouteEn author="emdoe" example="/nautil/topic/arts" path="/nautil/topic/:tid" :paramsDesc="['topic']">
```

Comment:
I would suggest continuing to use `nautilus` rather than `nautil`

Reply:
I know this is somehow a breaking change to users.
However, this helps reducing some unnecessary namespace collisions using purely the domain name instead of alias/site name, imagine someone wants an RSS feed for https://nautilus.org

Reply:
to me it's perfectly fine; however if users want feeds for _all_ three of these websites, the issue would persist. but thanks for the help!